testapi.pm

```diff
@@ -2250,4 +2257,59 @@ sub backend_get_wait_still_screen_on_here_doc_input () {
     return get_var(_WAIT_STILL_SCREEN_ON_HERE_DOC_INPUT => $ret);
 }
 
+=head2 get_username, set_username
+
+    my $un = get_username;
+    set_username($new_name);
+
+Wrapper around the exported C<$username> variable. Direct user of the variable
```

Comment:
"Direct user"?

Reply:
I meant "direct use". But I removed that now.
Not sure if/when we want to deprecate it at all.